ts/Series/FlowMap/FlowMapSeries.ts

```diff
@@ -0,0 +1,829 @@
+                toX - toXToArc,
+                toY - toYToArc
+            ], [ // The markerEnd will be spliced between these arrays.
+                'L',
+                toX + toXToArc,
+                toY + toYToArc
+            ], [
+                'Q',
+                arcPointX + wX,
+                arcPointY + wY,
+                fromX + fromXToArc,
+                fromY + fromYToArc
+            ], [
+                'Z'
+            ]]
+        };
+
+        if (markerEndOptions && markerEndOptions.enabled && shapeArgs.d) {
+            const marker: SVGPath = FlowMapSeries.markerEndPath(
+                [toX - toXToArc, toY - toYToArc],
+                [toX + toXToArc, toY + toYToArc],
+                [toPoint.plotX as number, toPoint.plotY as number],
+                markerEndOptions
+            );
+
+            (shapeArgs.d as SVGPath).splice(2, 0,
+                ...marker
+            );
+        }
+
+        return shapeArgs;
+    }
+}
+
+/* *
+ *
+ *  Prototype properties
+ *
+ * */
+
+interface FlowMapSeries {
+    pointClass: typeof FlowMapPoint;
+}
+
+extend(FlowMapSeries.prototype, {
+    pointClass: FlowMapPoint,
+    // Make it work on zoom or pan.
+    useMapGeometry: true
+});
+
+/* *
+ *
+ *  Registry
+ *
+ * */
+declare module '../../Core/Series/SeriesType' {
+    interface SeriesTypeRegistry {
+        flowmap: typeof FlowMapSeries;
+    }
+}
+
+SeriesRegistry.registerSeriesType('flowmap', FlowMapSeries);
+
+/* *
+ *
+ *  Default export
+ *
+ * */
+
+export default FlowMapSeries;
+
+/* *
+ *
+ *  API options
+ *
+ * */
+
+/**
+ * A `flowmap` series. If the [type](#series.flowmap.type) option
+ * is not specified, it is inherited from [chart.type](#chart.type).
+ *
+ * @extends   series,plotOptions.flowmap
+ * @excluding allAreas, allowPointSelect, boostBlending, boostThreshold,
+ * borderColor, borderWidth, dashStyle, dataLabels, dragDrop, joinBy, mapData,
+ * negativeColor, onPoint, shadow, showCheckbox
+ * @product   highmaps
+ * @apioption series.flowmap
+ */
+
+/**
+ * An array of data points for the series. For the `flowmap` series
+ * type, points can be given in the following ways:
+ *
+ * 1.  An array of arrays with options as values. In this case,
+ *     the values correspond to `from, to, weight`. Example:
+ *     ```js
+ *     data: [
+ *         ["Point 1", "Point 2", 4],
+ *     ]
+ *     ```
+ *
+ * 2.  An array of objects with named values. The following snippet shows only a
+ *     few settings, see the complete options set below.
+ *
+ *     ```js
+ *         data: [{
+ *             from: "Point 1",
+ *             to: "Point 2",
+ *             curveFactor: 0.4,
+ *             weight: 5,
+ *             growTowards: true,
+ *             markerEnd: {
+ *                 enabled: true,
+ *                 height: 15,
+ *                 width: 8
+ *             }
+ *         }]
+ *     ```
+ *
+ * @type      {Array<number|null|*>}
+ * @excluding labelrank, outgoing
```

Comment:
Why `outgoing` is excluded? This looks like not needed and not yet implemented. Sankey leftover?

Reply:
I think `labelrank` also is not needed to exclude. When removed nothing has changed in API.